src/main/kotlin/com/github/biomejs/intellijbiome/settings/BiomeOnSaveInfoProvider.kt

```diff
@@ -0,0 +1,24 @@
+package com.github.biomejs.intellijbiome.settings
+
+import com.github.biomejs.intellijbiome.BiomeBundle
+import com.intellij.ide.actionsOnSave.ActionOnSaveContext
+import com.intellij.ide.actionsOnSave.ActionOnSaveInfo
+import com.intellij.ide.actionsOnSave.ActionOnSaveInfoProvider
+
+class BiomeOnSaveInfoProvider : ActionOnSaveInfoProvider() {
+    override fun getActionOnSaveInfos(context: ActionOnSaveContext):
+        List<ActionOnSaveInfo> = listOf(
+        BiomeOnSaveFormatActionInfo(context),
+        BiomeOnSaveApplySafeFixesActionInfo(context),
+        BiomeOnSaveApplyUnsafeFixesActionInfo(context)
+    )
+
+    override fun getSearchableOptions(): Collection<String> {
+        return listOf(
+            BiomeBundle.message("biome.run.format.on.save.checkbox.on.actions.on.save.page"),
```

Comment:
Out of curiosity, does the IDE give you autocomplete when you try to write a label?

Reply:
Like this?
![CleanShot 2023-12-20 at 16 53 03](https://github.com/biomejs/biome-intellij/assets/78874691/429cb04c-1756-4786-8eaf-b39f19d68430)

Reply:
Nice! Yeah that's exactly what I meant